README.rst

```diff
@@ -21,7 +21,7 @@ Example `.travis.yml` file for testing Ansible roles:
       - docker
 
     install:
-      - pip install ome-ansible-molecule
+      - pip install "ome-ansible-molecule<0.5"
```

Comment:
Makes me wonder whether we should have released it as `ome-ansible-molecule 1.0.0` and really capped on major releases

Reply:
I'd support this proposal